apps/dot-cdn/src/app/app.component.scss

```diff
@@ -0,0 +1,103 @@
+@use "variables" as *;
+
+:host {
+    background: #f1f3f4; // temporary
+    display: block;
+    box-sizing: border-box;
+    padding: $spacing-3 0;
+}
+
+h1 {
+    margin: 0;
+}
+
+.dot-cdn__container {
+    box-shadow: $md-shadow-1;
+    background: $white;
+    margin: $spacing-3 $spacing-5;
+    display: block;
+}
+
+.dot-cdn {
+    &__tab-content {
+        padding: $spacing-4;
+
+        &--contained {
+            padding: $spacing-4;
+            textarea {
+                width: 100%;
+                margin-bottom: $spacing-4;
+            }
+        }
+    }
+}
```

Comment:
We used to use this a lot but now I think is a bad approach because when you try to search for classes in the codebase you typically get the class from the dom and you search for like `dot-cdn__tab-content` and you can't find it so is hard to find classes. We should stop doing this.